### DISCUSSION TITLE
Vercel Edge Config has a read limit of 50,000 in my project

### DISCUSSION THREAD

**Discussion by Doarakko** [vercel/community#7333] [Account Issue]
### Question

I was unable to contact you because I am on a hobby plan, so I will contact you here.
The documentation says the read limit for Vercel Edge config is 100,000, but on my dashboard it was 50,000.

![](https://github.com/vercel/community/assets/21009186/979bc320-d5a1-4be2-89b2-586137cc15a8)

document
https://vercel.com/docs/storage/edge-config/edge-config-limits

![](https://github.com/vercel/community/assets/21009186/40eb4a50-6c85-4803-9fc6-53dd148b16eb)

When I checked the release notes for Vercel Edge Config, the limit was 50,000. Does the limit depend on when the project was created?
Or has the pricing plans changed?

> Hobby users can perform 50,000 reads per month
https://vercel.com/blog/vercel-edge-config-is-now-generally-available

I always enjoy developing applications using Vercel, thank you!

**Comment by pawlean** [comment]
Hi @Doarakko!

Thanks for this post - doubling checking internally for you and will get back to you shortly. 😄

[a comment by an automated account is not shown]
